scripts/linters/pylint_extensions.py

```diff
@@ -141,9 +148,11 @@ def visit_call(self, node):
         num_positional_args_unused = num_positional_args
         # Check that all parameters with a default value have
         # been called explicitly.
+        keyword_args_in_funcdef = []
```

Comment:
Also, write a comment above this explaining what this variable will hold.

Reply:
Done.